pkg/signature/payload/payload_test.go

```diff
@@ -75,6 +75,14 @@ func TestMarshalCosign(t *testing.T) {
 			},
 			expected: `{"critical":{"identity":{"docker-reference":"example.com/cosign/test/image"},"image":{"docker-manifest-digest":"sha256:d34db33fd34db33fd34db33fd34db33fd34db33fd34db33fd34db33fd34db33f"},"type":"cosign container image signature"},"optional":{"CamelCase WithSpace":8.314,"creator":"anyone","some_struct":{"false":true,"foo":"bar","nothing":null}}}`,
 		},
+		{
+			desc: "custom identity",
+			imgPayload: Cosign{
+				Image:    mustParseDigest(t, "example.com/test/image@"+validDigest),
+				Identity: "test",
```

Comment:
Non-blocking:
```suggestion
				Identity: "docker.io/library/test:1.2.3",
```
and likewise please use tags in the other ones, to be a bit more plausible as examples (fully-qualified, and with tag values) — especially now that these are the first/only actual uses in this repo.